### ISSUE TITLE
Inference

### ISSUE THREAD

**Issue by Sreyan88** [s3prl/s3prl#77]
Hi there,

I have successfully finetuned pre-trained mockingjay for downstream mosei task using:

`python run_downstream.py -m train -u mockingjay --upstream_trainable --downstream mosei --expname first`

I have my checkpoints saved in results/downstream/first.

I wanted to know how I can use the same for getting sentiment probabilities using s3prl? I want to get probabilities for all of my original train/test/dev and a new test folder having audio files too.

Please let me know of the procedure or whether I have to load checkpoint manually in python and do inference?

Thank You

**Comment by leo19941227** [comment]
Hi,

You can use `python run_downstream.py -m evaluate -e [ckpt_path]` to re-evaluate your saved checkpoint using the `evaluate` method defined in `downstream/runner.py`, and in that procedure you can save the logits.

While this command is currently only for testing set, you can modify the default [split](https://github.com/s3prl/s3prl/blob/e59a5c5c6dfcb77a33d1e064b72b587b4e10ba04/downstream/runner.py#L277) argument to get the results on train/dev set.

**Comment by Sreyan88** [comment]
Found it !Thank You for the answer as always!
